packages/layout/src/tree-layout/data-adapters/data-point.js

```diff
@@ -0,0 +1,12 @@
+export class DataPoint {
+    constructor (node) {
+        this.top = node.boundBox.top;
```

Comment:
Use methods to get and set variables.

Reply:
If a variable or object is referenced multiple times, store it in a variable and use it. Also, try to use destructuring

for example, use const {top, left, width, height }=  node.boundBox;